db/migrate/20231110153746_create_todo_table.rb

```diff
@@ -0,0 +1,10 @@
+class CreateTodoTable < ActiveRecord::Migration[6.0]
+  def change
+    create_table :todos do |t|
```

Comment:
Consider adding a database index on the 'title' column in the 'todos' table to improve query performance, especially if this field will be frequently searched or filtered.